api-operator/deploy/controller-artifacts/operator.yaml

```diff
@@ -32,7 +32,7 @@ spec:
       containers:
         - name: api-operator
           # Replace this with the built image name
-          image: wso2/k8s-api-operator:1.2.2
+          image: jayanieb/envoy-k8s-api-operator:v1.0.0
```

Comment:
shouldn't this point to wso2 account?

Reply:
same with `cachev1alpha1 "github.com/Shehanir/k8s-securityOperator/security-operator/pkg/apis/cache/v1alpha2"`

Reply:
This is fixed in latest